src/SeleniumTestCase.php

```diff
@@ -3,51 +3,65 @@
 namespace Modelizer\Selenium;
 
 use Modelizer\Selenium\Services\Application as Laravel;
+use Modelizer\Selenium\Services\InteractsWithPage as Interactions;
+use Modelizer\Selenium\Services\WaitsForElements;
+use Modelizer\Selenium\Services\WorksWithDatabases;
 use PHPUnit_Extensions_Selenium2TestCase;
 
 class SeleniumTestCase extends PHPUnit_Extensions_Selenium2TestCase
 {
-    use Laravel;
+    use Laravel,
+        Interactions,
+        WorksWithDatabases,
+        WaitsForElements;
+
+    /**
+     * @var string
+     */
+    protected $baseUrl;
 
     protected function setUp()
     {
         $this->setUpLaravel();
-
-        $this->setBrowserUrl(env('APP_URL', 'http://localhost/'));
+        $this->baseUrl = env('APP_URL', 'http://localhost/');
+        $this->setBrowserUrl($this->baseUrl);
         $this->setBrowser(env('DEFAULT_BROWSER', 'chrome'));
     }
 
-    protected function visit($path)
-    {
-        $this->url($path);
-
-        return $this;
-    }
-
-    protected function see($text, $tag = 'html')
+    public function setupPage()
     {
-        $this->assertContains($text, $this->byTag($tag)->text());
-
-        return $this;
+        if (!empty(getenv('SELENIUM_WIDTH')) && !empty(getenv('SELENIUM_HEIGHT')) &&
```

Comment:
Ok,

This is updated to allow both essentially.

You can set the window width and height on your test directly through

```php
class TestTest extends SeleniumTestCase {
    
    $this->width = 1024;
    $this->height = 768;
}
```

or within your .env file with:

```php
SELENIUM_WIDTH=1024
SELENIUM_HEIGHT=768
```

And you can also do things within your test like:

```php

class TestTest extends SeleniumTestCase {
    
    /**
     * @test
     **/
    public function it_can_resize_a_window_width()
    {
          $this->setWidth(1024);
    }

    /**
     * @test
     **/
    public function it_can_resize_a_window_height()
    {
          $this->setHeight(768);
    }

    /**
     * @test
     **/
    public function it_can_resize_a_window_in_both_dimensions()
    {
          $this->changeWindowSize(1024, 768);
    }

}
```